app/models/linkage_group.rb

```diff
@@ -20,5 +19,41 @@ class LinkageGroup < ActiveRecord::Base
   validates :consensus_group_assignment,
             presence: true
 
+  include Relatable
+  include Filterable
+  include Pluckable
+
+  def self.table_data(params = nil)
+    query = (params && params[:query].present?) ? filter(params) : all
+    query.pluck_columns
+  end
+
+  def self.table_columns
+    [
+      'linkage_group_label',
+      'linkage_group_name',
+      'total_length',
+      'lod_threshold',
+      'consensus_group_assignment',
+      'consensus_group_orientation'
+    ]
+  end
+
+  def self.count_columns
+    [
+      'linkage_groups.map_linkage_group_lists_count AS linkage_maps_count'
+    ]
+  end
+
+  private
+
+  def self.permitted_params
```

Comment:
That's what I was talking a few days ago - it does not matter if you put `self.permitted_params` below `private` or not. It is always public.